pom.xml

```diff
@@ -76,14 +76,14 @@
 
   <properties>
     <awaitility.version>3.1.6</awaitility.version>
-    <cdap.version>6.8.0-SNAPSHOT</cdap.version>
```

Comment:
why are making these changes

Reply:
When using the snapshot version , some dependency appear to be missing , hence the plugin won't build.

I am not sure how how to bump the version, for now removing the snapshot version appears to fix missing dependency.